python/pyarrow/src/arrow/python/numpy_to_arrow.cc

```diff
@@ -62,7 +62,7 @@
 namespace arrow {
 
 using internal::checked_cast;
-using internal::CopyBitmap;
+// using internal::CopyBitmap;
```

Comment:
Note to self: this symbol is not resolving when linking against libarrow. Inspecting the mangled names, it appears that the default argument value of 0 is the culprit. Need to still figure out why that is problematic